.github/workflows/ci.yml

```diff
@@ -52,11 +52,24 @@ jobs:
       - name: cargo deny check
         run: cargo deny check
 
-  # ──────────────────────── 2 · GCS feature test ────────────────────────────
+  # ──────────────────────── 2 · GCS test (feature, chaos) ─────────────────
   gcs_test:
-    name: Unit tests with GCS feature enabled
+    name: GCS tests (feature, chaos)
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        type: [feature, chaos]
+        include:
+          - type: feature
+            name: GCS feature test
+            timeout-minutes: 5
+            cmd: |
+              cargo test -p moonlink --features=storage-gcs
+          - type: chaos
+            name: GCS chaos test
+            timeout-minutes: 10
+            cmd: |
+              RUST_BACKTRACE=1 cargo test table_handler::chaos_gcs_test --features=chaos-test,storage-gcs -p moonlink -- --nocapture
```

Comment:
https://github.com/Mooncake-Labs/moonlink/actions/runs/16636664570/job/47079294460?pr=1148
GCS chaos test doesn't seem to run anything

Reply:
Could you please do me a favor, to switch `table_handler::chaos_gcs_test` to `test_gcs_chaos`